lib/repl.js

```diff
@@ -844,10 +844,49 @@ REPLServer.prototype.createContext = function() {
   });
 
   addBuiltinLibsToObject(context);
+  previewResults.call(this);
 
   return context;
 };
 
+function previewResults() {
+  this.on('buffer', function(line) {
+
+    Interface.prototype.clearPreview.call(this);
+
+    // no need of preview for a multiline statement
+    if (this[kBufferedCommandSymbol] !== '')
+      return;
+
+    sendInspectorCommand((session) => {
+      session.post('Runtime.evaluate', {
+        expression: line.toString(),
+        generatePreview: true,
+        throwOnSideEffect: true,
+        timeout: 500,
+        executionContextId: this[kContextId]
+      }, (error, previewResult) => {
+
+        if (error) {
+          debug(`Error while generating preview ${error}`);
+        }
+
+        if (undefined !== previewResult.result.value) {
+          const value = util.inspect(previewResult.result.value);
+          Interface.prototype.appendPreview.call(this, value);
+        }
+
+        if (previewResult.result.preview) {
+          const previewResults = previewResult.result.preview.properties;
+          const previewData = previewResults.map((preview) => preview.value); // eslint-disable-line max-len
```

Comment:
I think this should avoid the eslint message:
```suggestion
          const previewData = previewResults.map(
            (preview) => preview.value
          );
```